js/forms.js

```diff
@@ -279,85 +279,181 @@
      * Auto complete plugin  *
      *************************/
     $.fn.autocomplete = function (options) {
-      // Defaults
       var defaults = {
-        data: {}
-      };
+        data: {},
+        maxElementsAmount: null
+      },
+      currentLi = 0,
+      autocompleteOption = null,
+      currentAutocompleteVal = '';
 
       options = $.extend(defaults, options);
 
-      return this.each(function() {
-        var $input = $(this);
-        var data = options.data,
-            $inputDiv = $input.closest('.input-field'); // Div to append on
+      // Set input value
```

Comment:
The JavaDoc doesn't give more information than already hinted by the name of the function.

Is inputElement the main $input? Why is the parameter needed? Can't you just use $input captured in the closure?

Why is this function needed? It's only used once in the code. I think just remove the function and put its body where the call on line 450 is.